src/pages/Items/BestProductList.tsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import BestProductItem from "./BestProductItem";
+import { getProductItem } from "./api";
+
+interface BestProductListProps {
+  pageSize: number;
+}
+
+interface Product {
+  createdAt: string;
+  description: string;
+  favoriteCount: number;
+  id: number;
+  images: string[];
+  name: string;
+  ownerId: number;
+  price: number;
+  tags: string[];
+  updatedAt: string;
+}
+
+const BestProductList = ({ pageSize }: BestProductListProps) => {
+  const [product, setProduct] = useState<Product[]>([
+    {
+      createdAt: "",
+      description: "",
+      favoriteCount: 0,
+      id: 0,
+      images: [],
+      name: "",
+      ownerId: 0,
+      price: 0,
+      tags: [],
+      updatedAt: "",
+    },
+  ]);
+
+  useEffect(() => {
+    const getBestProduct = async () => {
+      const data = await getProductItem(1, pageSize, "favorite");
+      setProduct(data.list);
+    };
+
+    getBestProduct();
+  }, [pageSize]);
+
+  return (
+    <section className="best-product">
+      <h2>베스트 상품</h2>
+      <ul className="item-list">
+        {product.map((item) => (
+          <li key={item.id}>
+            <Link to={`./${item.id}`} className="item-link">
+              <BestProductItem
+                imgUrl={item.images[0]}
+                name={item.name}
+                price={item.price}
+                favoriteCount={item.favoriteCount}
+              />
+            </Link>
+          </li>
+        ))}
```

Comment:
item 객체 안에 있는 내용을 반복적으로 가져와 사용하고 있으므로 
콜백함수의 매개변수 부분에서 구조분해 할당을 사용한다면 더욱 간결하게 작성할 수 있을 것 같아요. 😄

```suggestion
        {product.map(({ id, images, name, price, favoriteCount }) => (
          <li key={id}>
            <Link to={`./${id}`} className="item-link">
              <BestProductItem
                imgUrl={images[0]}
                name={name}
                price={price}
                favoriteCount={favoriteCount}
              />
            </Link>
          </li>
        ))}
```